draft-tus-resumable-uploads-protocol.md

```diff
@@ -175,16 +175,14 @@ Client                                      Server
 
 # Upload Transfer Procedure {#upload-transfer}
 
-The Upload Transfer Procedure can be used for either starting a new upload, or resuming an existing upload. A limited form of this procedure MAY be used by the client to start a new upload without the knowledge of server support.
+The Upload Transfer Procedure is intended for transferring the data chunk. As such, it can be used for either resuming an existing upload, or starting a new upload. A limited form of this procedure MAY be used by the client to start a new upload without the knowledge of server support.
 
-This procedure is designed to be compatible with a regular upload. Therefore all methods are allowed with the exception of `GET`, `HEAD`, `DELETE`, and `OPTIONS`. And all response status codes are allowed. The client is RECOMMENDED to use `POST` request if not otherwise specified.
+This procedure is designed to be compatible with a regular upload. Therefore all methods are allowed with the exception of `GET`, `HEAD`, `DELETE`, and `OPTIONS`. And all response status codes are allowed. The client is RECOMMENDED to use the `POST` method if not otherwise intended. The server MAY only support a limited number of methods.
 
```

Comment:
Minor: "And" is probably not needed here? `And all response status codes` -> `All response status codes`